clients/src/main/java/org/apache/kafka/common/network/KafkaChannel.java

```diff
@@ -129,6 +133,7 @@ public void prepare() throws AuthenticationException, IOException {
             // Clients are notified of authentication exceptions to enable operations to be terminated
             // without retries. Other errors are handled as network exceptions in Selector.
             state = new ChannelState(ChannelState.State.AUTHENTICATION_FAILED, e);
+            muteOnPrepareFailure();
```

Comment:
We don't want to do this on the client side since we are delaying only on the broker?